src/tasks/python/PythonExtensionHelper.ts

```diff
@@ -19,6 +17,23 @@ export namespace PythonExtensionHelper {
     }
 
     export async function getLauncherFolderPath(): Promise<string> {
+        const pyExt = await getPythonExtension();
+
+        /* eslint-disable @typescript-eslint/tslint/config */
+        if (pyExt?.exports?.debug) {
+            const debuggerPath = await pyExt.exports.debug.getDebuggerPackagePath();
+
+            if (debuggerPath) {
+                return debuggerPath;
+            }
+        }
+        /* eslint-enable @typescript-eslint/tslint/config */
+
+        throw new Error(localize('vscode-docker.tasks.pythonExt.noDebugger', 'Unable to find the debugger in the Python extension.'));
+    }
+
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    export async function getPythonExtension(): Promise<vscode.Extension<any>> | undefined {
         const pyExtensionId = 'ms-python.python';
         const minPyExtensionVersion = new semver.SemVer('2020.5.78807');
```

Comment:
Shouldn't there be a new minimum version?

Reply:
No, the API already exists in the current minimum version (based on check-in dates at least). I can quickly verify that with Karthik.